web/templates/draft_table_body.hbs

```diff
@@ -6,10 +6,23 @@
                 <div class="exit">
                     <span class="exit-sign">&times;</span>
                 </div>
-                <div class="removed-drafts">
-                    {{t "Drafts are not synced to other devices and browsers." }}
-                    <br />
-                    {{#tr}}Drafts older than <strong>{draft_lifetime}</strong> days are automatically removed.{{/tr}}
+                <div class="header-body">
+                    <div class="removed-drafts">
```

Comment:
Let's change this class to something like `removed-drafts-message` or something. As it stands, it sounds like this class marks drafts that have been removed (which of course is impossible, but future contributors will thank you).

Reply:
Yes, that makes perfect sense to me.